### ISSUE TITLE
error:Cound not get branch revision

### ISSUE THREAD

**Issue by token-cjg** [godotengine/webrtc-native#17]
### TLDR

`./build.sh -c x64 -b branch-heads/68` no longer works properly in the repo and spits out the above error, when building on a mac.

### Context

I've been trying to build webrtc for mac.  Notwithstanding the fact that this has been done already in this issue: https://github.com/godotengine/webrtc-native/issues/8, I only noticed that after doing some investigation that led me down a particular path.

In https://github.com/godotengine/webrtc-native/issues/4, I encountered the same problem as shintiger.  After managing to get things to build for the mac platform, I came across the error in this issue `fatal error: api/peerconnectioninterface.h: No such file or directory`, then searched the issues for this project and came across [said thread](https://github.com/godotengine/webrtc-native/issues/4).

### Hypothesis

Some amateur sleuthing for the above error suggests that it looks like the old way of getting Chromium code [has been obsoleted](https://sites.google.com/a/chromium.org/dev/developers/how-tos/old-get-the-code).  This uses things that look familiar from this codebase, like ninga, gclient, out directories, etc.

Therefore it appears that scripts will need to be updated to get the Chromium code [using this new approach](https://chromium.googlesource.com/chromium/src/+/master/docs/mac_build_instructions.md).

I'm not sure how that impacts getting things like branch 68
https://chromium.googlesource.com/external/webrtc/+/branch-heads/68 via the invocation `./build.sh -c x64 -b branch-heads/68`, but certainly the latter command no longer works with the change in the way Chromium code gets fetched.

**Comment by Faless** [comment]
The new [`0.5 release`](https://github.com/godotengine/webrtc-native/releases/tag/0.5) has been updated to use WebRTC `branch-heads/4472`. Closing as fixed.
